core/slice/slice.odin

```diff
@@ -117,6 +131,61 @@ linear_search_proc :: proc(array: $A/[]$T, f: proc(T) -> bool) -> (index: int, f
 	return -1, false
 }
 
+/*
+	Reverse linear search searches the given element in the given slice in O(n) time,
+	starting from the slice end.
+
+	Returns the last index at which the given element can be found in the slice,
+	or -1 if it is not present
+
+	# Examples
```

Comment:
Examples should be formatted as described here: https://github.com/odin-lang/pkg.odin-lang.org?tab=readme-ov-file#markup-rules